iothub/service/src/Exceptions/IotHubSuspendedException.cs

```diff
@@ -14,13 +14,15 @@ namespace Microsoft.Azure.Devices.Common.Exceptions
     [Serializable]
     public class IotHubSuspendedException : IotHubException
     {
+        private const string SuspendedException = "IotHubSuspendedException";
+
         /// <summary>
         /// Creates an instance of this class with a name of the suspended IoT hub
         /// and marks it as non-transient.
         /// </summary>
         /// <param name="iotHubName">The name of the IoT hub that has been suspended.</param>
         public IotHubSuspendedException(string iotHubName)
-            : base(Resources.IotHubSuspendedException.FormatInvariant(iotHubName))
+            : base("Iothub {0} is suspended".FormatInvariant(iotHubName))
```

Comment:
```suggestion
            : base("IoT hub {0} is suspended".FormatInvariant(iotHubName))
```

Reply:
Added the change.
https://github.com/Azure/azure-iot-sdk-csharp/pull/2621/commits/92916289035d8317494212bd4193347d5f7f1cda